src/impact-dashboard/CurveChartLegend.tsx

```diff
@@ -15,19 +15,49 @@ const LegendItem = styled.div`
   margin: 0 1em;
   font-weight: normal;
   padding-bottom: 0.6em;
+  cursor: pointer;
 `;
 
 interface Props {
   markColors: MarkColors;
+  toggleGroup: Function;
+  groupStatus: {
+    [propName: string]: boolean;
+  };
 }
 
-const CurveChartLegend: React.FC<Props> = ({ markColors }) => {
+const CurveChartLegend: React.FC<Props> = ({
+  markColors,
+  toggleGroup,
+  groupStatus,
+}) => {
+  () => alert("hello");
   return (
     <LegendWrapper>
-      <LegendItem color={markColors.exposed}>exposed</LegendItem>
-      <LegendItem color={markColors.infectious}>infectious</LegendItem>
-      <LegendItem color={markColors.hospitalized}>hospitalized</LegendItem>
-      <LegendItem color={markColors.fatalities}>fatalities</LegendItem>
+      <LegendItem
+        onClick={() => toggleGroup("exposed")}
+        color={markColors.exposed}
+      >
+        {groupStatus["exposed"] ? "exposed" : "exposed"}
```

Comment:
this logic (and similar statements for the other items) isn't doing anything, you can just render the string directly

Reply:
What's the idea here, @AhmedMugtaba? Why have a ternary operator that has the same value on both sides?

Reply:
This was meant to be a place holder to implement Active/Inactive functionality on legend when the user toggled, will update today.